fieldservice_person_salary/reports/fsm_order_template.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template
+        id="report_fsm_order_document_person_salary"
+        inherit_id="fieldservice.report_fsm_order_document"
+    >
+
+        <xpath expr="//div[@id='description']" position="after">
+            <div id="planning">
```

Comment:
An optional improvement should be hide the whole section in the report if all the fields of the sections are unset. Please comment this with project manager LR, I'm not sure what is the best option

Reply:
At the moment it is not necessary according to what he tells me